Makefile

```diff
@@ -70,10 +70,10 @@ check_shebangs:
 check_trailing_whitespace:
 	@source test/make.sh && check_trailing_whitespace
 
-# .PHONY: check_headers
-# check_headers:
-# 	@echo "Checking file headers"
-# 	@python test/verify_boilerplate.py
+.PHONY: check_headers
+check_headers:
+	@python hack/boilerplate/boilerplate.py
```

Comment:
Please keep the name 'test' for the directory.  As a group it was decided to use 'test' instead of 'hack' :)